test/service/test_drs.py

```diff
@@ -140,46 +141,62 @@ def test_drs(self):
                         # We expect a DRS object with an access URL
                         expected['access_methods'] = [
                             {
-                                'access_url': {
-                                    'url': ('https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
-                                            'blobs/307.a72.eb6?foo=bar&et=cetera')
-                                },
+                                'access_url': {'url': ('https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
```

Comment:
What's the reason for the parentheses here? String continuation works without them and is less easy to confuse with a tuple.

Reply:
There's no reason. removed :+1: